apps/mobile/src/features/approver/components/approver-permissions.tsx

```diff
@@ -0,0 +1,45 @@
+import { t } from '@lingui/macro';
+
+import { Box, Cell, CheckmarkIcon, Text } from '@leather.io/ui/native';
+
+type Permissions = 'view_balance_activity' | 'request_approval';
+
+interface ApproverPermissionsProps {
+  permissions: Permissions[];
+}
+
+export function ApproverPermissions({ permissions }: ApproverPermissionsProps) {
```

Comment:
Does this component need to be in any way associated to Approver? Why not just PermissionsList or something, outside Approver feature